recipes/confindr/meta.yaml

```diff
@@ -11,25 +11,23 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
   script: python -m pip install --no-deps --ignore-installed --no-cache-dir .
 
 requirements:
   host:
-    - python >=3
+    - python >=3.9.15
     - pip
   run:
-    - python >=3
-    - biopython
-    - rauth
+    - python >=3.9.15
+    - biopython 1.81
```

Comment:
That looks like a too specific pin. Is that really needed?

Reply:
Hi Björn, yes Biopython 1.81 is necessary.